src/plugins/saved_objects_management/public/management_section/objects_table/__snapshots__/saved_objects_table.test.tsx.snap

```diff
@@ -278,6 +278,9 @@ exports[`SavedObjectsTable import should show the flyout 1`] = `
         },
       },
       "search": [MockFunction],
+      "searchParams": Object {
+        "createFromRequest": [MockFunction],
+      },
```

Comment:
That's not the first PR that is forced to update this snapshot. This should not be needed. Gonna fix the test to just assert existence of the `Flyout` component. Snapshot testing the props has really low value here, and behavior is already covered in FTR test anyway

Reply:
Created https://github.com/elastic/kibana/pull/68489